insertion.py

```diff
@@ -0,0 +1,19 @@
+import time
+import random
+
+def tri_insertion(table: list) -> list:
+    table_length: int = len(table)
+    #On parcours le tableau de 0 à n dans la boucle externe & de l'index du tableau 1 à 0.
+    for i in range(table_length):
+        for j in range(i,0,-1):
```

Comment:
En effet l'autre version est meilleure car cette boucle for va continuer même quand on aura dépassé l'élément qui nous intéresse

Reply:
C'est bien d'avoir fait les deux !